bitnami/solr/README.md

```diff
@@ -0,0 +1,334 @@
+| `commonLabels`                          | Common lables to add to the Kuberentes objects                                                                                                        | `{}`                                                      |
+| `commonAnnotations`                     | Common annotations to add to the Kubernetes objects                                                                                                   |                                                           |
+| `extraDeploy`                           | Array of extra objects to deploy with the release (evaluated as a template).                                                                          | `nil`                                                     |
+| `replicaCount`                          | Number of solr replicas                                                                                                                               | `{}`                                                      |
+| `image.registry`                        | Solr Image registry                                                                                                                                   | `docker.io`                                               |
+| `image.repository`                      | Solr Image name                                                                                                                                       | `bitnami/redis`                                           |
+| `image.tag`                             | Solr Image tag                                                                                                                                        | `{TAG_NAME}`                                              |
+| `image.pullPolicy`                      | Image pull policy                                                                                                                                     | `IfNotPresent`                                            |
+| `image.pullSecrets`                     | Specify docker-registry secret names as an array                                                                                                      | `nil`                                                     |
+| `podSecurityContext.fsGroup`            | Group ID for the pods.                                                                                                                                | `1001`                                                    |
+| `podSecurityContext.sysctls`            | Set namespaced sysctls for the pods.                                                                                                                  | `nil`                                                     |
```

Comment:
Ran a check and saw these values defined in values.yaml not defined in the README

```
ERROR: Value cloudBootstrap is not in README.md
ERROR: Value podLabels is not in README.md
ERROR: Value podAnnotations is not in README.md
ERROR: Value lifecycleHooks is not in README.md
ERROR: Value tls.enabled is not in README.md
ERROR: Value tls.wantClientAuth is not in README.md
ERROR: Value tls.needClientAuth is not in README.md
ERROR: Value tls.keystorePassword is not in README.md
ERROR: Value tls.importKubernetesCA is not in README.md
ERROR: Value tls.checkPeerName is not in README.md
ERROR: Value tls.caSecret.name is not in README.md
ERROR: Value tls.caSecret.bundlePath is not in README.md
ERROR: Value tls.certSecret.name is not in README.md
ERROR: Value tls.certSecret.keyPath is not in README.md
ERROR: Value tls.certSecret.certPath is not in README.md
ERROR: Value persistence.annotations is not in README.md
ERROR: Value persistence.mountPath is not in README.md
ERROR: Value zookeeper.service.publishNotReadyAddresses is not in README.md
ERROR: Value exporter.port is not in README.md
ERROR: Value exporter.threads is not in README.md
ERROR: Value exporter.containerSecurityContext is not in README.md
ERROR: Value exporter.podSecurityContext is not in README.md
ERROR: Value exporter.updateStrategy is not in README.md
ERROR: Value exporter.service.port is not in README.md
ERROR: Value exporter.service.nodePorts.http is not in README.md
ERROR: Value exporter.service.nodePorts.https is not in README.md
ERROR: Value exporter.service.externalTrafficPolicy is not in README.md
ERROR: Value exporter.service.loadBalancerSourceRanges is not in README.md
```